static/styles/draft_results.css

```diff
@@ -0,0 +1,29 @@
+table {
```

Comment:
pretty highly recommend avoiding putting styles directly on the html tags. 

would be good for us to introduce classes and then apply that class to the relevant tags